test/arrays/userAPI/enumArray.chpl

```diff
@@ -0,0 +1,12 @@
+use arrayAPItest;
+
+enum color {red=42, orange=33, yellow=31, green=22, blue=8, indigo=5, violet=3};
```

Comment:
Did you intend for the scrambling to result in a strictly descending sequence?

(It still passes when I change a few of the values to make the sequence non-monotonic.)

Reply:
Yeah.  I consider this whitebox testing, where I knew the code well enough to feel confident it would work (and felt that other tests covered other cases well enough that something would go wrong if it broke).